Formula/grass.rb

```diff
@@ -1,13 +1,13 @@
 class Grass < Formula
   desc "Geographic Resources Analysis Support System"
   homepage "https://grass.osgeo.org/"
-  revision 1
```

Comment:
This should either be left as-is, or changed to `revision 2` if everyone should receive the updated version.